keystrokes/src/components/Character/CharacterList.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Typically I like to group my imports into 2-3 types 
top being react, other libraries e.g. lodash or date-functions or prop-types. 
middle being still libraries but more components e.g. material stuff 
below any custom components and then at the bottom configs, style sheets, constants  

You don't have to follow that specifically, but i'd suggest grouping your imports in a way that makes sense to you